org.freedesktop.Platform.ffmpeg.json

```diff
@@ -58,14 +58,15 @@
                 "--enable-decoder=pcm_u8,pcm_s16le,pcm_s24le,pcm_f32le",
                 "--enable-decoder=theora,vorbis,vp8,vp9,mp3,flac,webp",
                 "--enable-decoder=rawvideo,png,gif",
+                "--enable-hwaccel=vp9_vaapi",
```

Comment:
Since this isn't encumbered this should be matched here: https://github.com/flatpak/freedesktop-sdk-images/blob/1.6/org.freedesktop.Sdk.json.in#L2085

Reply:
Agreed that's why I tagged @alexlarsson on #1, let's try again :)
I'll send a PR myself otherwise.

Reply:
Ah, yea a PR is probably reasonable, he is offline on the weekends.